tools/db_seed.py

```diff
@@ -0,0 +1,138 @@
+import os
+import subprocess
+from pathlib import Path
+import signal
+import requests
+
+import tdtax
+from social_tornado.models import TornadoStorage
+
+from baselayer.app.env import load_env
+from baselayer.app.model_util import status, create_tables
+from skyportal.models import Base, init_db, DBSession, User
+from skyportal.model_util import setup_permissions, create_token
+from skyportal.tests import api
+from baselayer.tools.test_frontend import verify_server_availability
+
+
+if __name__ == "__main__":
```

Comment:
The logic here is alsmot identical to that in `make load_demo_data`. Might make sense to factor out the common code into a single place, have `load_demo_data.py` and `db_seed.py` import what is needed?